google-account-plugin/resources/messages/AccountBundle.properties

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2016 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+login.toolbar.button.sign.in.text="Sign in to Google..."
+
+login.panel.sign.in.button.text=Sign In
+login.panel.sign.out.button.text=Sign Out
+login.panel.add.account.button.text=Add Account
+login.panel.play.console.link.text=Open Google Developers Console
+login.panel.cloud.console.link.text=Open Play Developer Console
+login.panel.play.console.link.url=https://play.google.com/apps/publish/#ProfilePlace
+login.panel.cloud.console.link.url=https://console.developers.google.com/
```

Comment:
I was thinking about the translators too and was wondering if they would potentially point to a different URLs. But if that's not a concern, I'll just remove those.